training_scripts/gsgnn_lp/ml_lm_input_lp.yaml

```diff
@@ -0,0 +1,49 @@
+---
+version: 1.0
+lm_model:
+  node_lm_models:
+    -
+      lm_type: bert
+      model_name: "bert-base-uncased"
+      gradient_checkpoint: true
+      node_types:
+        - movie
+gsf:
+  basic:
+    backend: gloo
+    ip_config: ip_list.txt
+    part_config: movielen_100k_lp_user_feat_movie_token_1p/ml.json
+    verbose: false
+    save_perf_results_path: null
+  gnn:
```

Comment:
this is confusing. it says gnn, but the encoder is mlp.

Reply:
changed